Feature 1 is almost done!

DaisyLi233 opened the pull request:
I've connected my part to Kenneth's part! So what's going on is after each person logs in, an output data structure from log in system has been created, and inside that output data structure it has two static variables so that they can be used directly. One of the variables inside output data structure is a boolean value, true if the person is a manager false if not.
I've also tested the case the person logged in is not a manager and when I tried to update the price, a proper error message shows up! That's great!
So far, I think my use case works well :)

Commit:
Because of the change of the item entity, I have to modify my test accordingly.

Commit:
Now my part is connected to Kenneth's part, and now the function which only manager can update the price is tested successfully!

Robinrswanson reviewed (commented):
Everything looks good!

Commit:
Fixed some problems for running tests, and changed the UserName Password dataset into the dataset we created before (before it was created by Kenneth for simple purposes).

DaisyLi233 commented:
Update version#2: Fixed some problems for running tests, and changed the UserName Password dataset into the dataset we created before (before it was created by Kenneth for simple purposes).

Commit:
Merge branch 'main' into feature-1

# Conflicts:
#	.gradle/7.4/executionHistory/executionHistory.bin
#	.gradle/7.4/executionHistory/executionHistory.lock
#	.gradle/7.4/fileHashes/fileHashes.bin
#	.gradle/7.4/fileHashes/fileHashes.lock
#	.gradle/7.4/fileHashes/resourceHashesCache.bin
#	.gradle/buildOutputCleanup/buildOutputCleanup.lock
#	.gradle/file-system.probe
#	build/classes/java/main/Main2.class
#	build/tmp/compileJava/previous-compilation-data.bin

Commit:
Since someone merged their codes, updated and committed changes.

Commit:
Since someone merged their codes, updated and committed changes.

Commit:
Since someone merged their codes, updated and committed changes.

Commit:
Since someone merged their codes, updated and committed changes.

Commit:
Since someone merged their codes, updated and committed changes.

tonyxieuoft commented:
Yep, everything looks good! Honestly, you could even make the Output Data Structure from Login a Singleton.

Commit:
Since someone merged their codes, updated and committed changes.

Commit:
Merge remote-tracking branch 'origin/main'

# Conflicts:
#	.gradle/7.4/executionHistory/executionHistory.bin
#	.gradle/7.4/executionHistory/executionHistory.lock
#	.gradle/7.4/fileHashes/fileHashes.bin
#	.gradle/7.4/fileHashes/fileHashes.lock
#	.gradle/7.4/fileHashes/resourceHashesCache.bin
#	.gradle/buildOutputCleanup/buildOutputCleanup.lock
#	.gradle/file-system.probe
#	build/classes/java/main/Main2.class
#	build/tmp/compileJava/previous-compilation-data.bin
#	build/tmp/compileTestJava/previous-compilation-data.bin

Commit:
Merge branch 'main' into feature-1

# Conflicts:
#	.gradle/7.4/executionHistory/executionHistory.bin
#	.gradle/7.4/executionHistory/executionHistory.lock
#	.gradle/7.4/fileHashes/fileHashes.bin
#	.gradle/7.4/fileHashes/fileHashes.lock
#	.gradle/7.4/fileHashes/resourceHashesCache.bin
#	.gradle/buildOutputCleanup/buildOutputCleanup.lock
#	build/classes/java/main/Main2.class
#	build/tmp/compileJava/previous-compilation-data.bin
#	build/tmp/compileTestJava/previous-compilation-data.bin
#	src/main/java/historydatabase/history.csv

Commit:
moved some files and rename

Commit:
haha!

Commit:
haha!

Commit:
Merge remote-tracking branch 'origin/feature-1' into feature-1

# Conflicts:
#	.gradle/7.4/executionHistory/executionHistory.bin
#	.gradle/7.4/executionHistory/executionHistory.lock
#	.gradle/7.4/fileHashes/fileHashes.bin
#	.gradle/7.4/fileHashes/fileHashes.lock
#	.gradle/7.4/fileHashes/resourceHashesCache.bin
#	.gradle/buildOutputCleanup/buildOutputCleanup.lock
#	.gradle/file-system.probe
#	build/classes/java/main/Main2.class
#	build/tmp/compileJava/previous-compilation-data.bin
#	build/tmp/compileTestJava/previous-compilation-data.bin
